External/TestDataSet/rawdata/sub-Sub1/perf/sub-Sub1_m0scan.json

```diff
@@ -0,0 +1,48 @@
+{
+	"Modality":"MR",
+	"MagneticFieldStrength":3,
+	"ImagingFrequency":127.744,
+	"Manufacturer":"Philips",
+	"ManufacturersModelName":"Ingenuity",
+	"SoftwareVersions":"3.2.2_3.2.2.1",
+	"MRAcquisitionType":"2D",
+	"ScanningSequence":"GR",
+	"SequenceVariant":"SK",
+	"ScanOptions":"FS",
+	"AcquisitionTime":[14,17],
+	"AcquisitionVoxelSize":[3.4,3.4,4.5],
+	"PhilipsRWVSlope":1.25543,
```

Comment:
This is an issue. I have send Paula an M0.nii with a correct scaling for BIDS? Do you have that one? If yes, then take the one from Paula and replace the M0.nii here with the one from her. Then simply run 
` jsonM0Write = xASL_bids_VendorFieldCheck(jsonM0Write);`
`                        jsonM0Write = xASL_bids_JsonCheck(jsonM0Write,'M0');`

If you don't have it, then we have to rescale it ourselves, and only then remove the Philips scale slopes.

Reply:
Are you talking to @BeatrizPadrela or to me? :D 
I don't know Paula and I don't have any clue about the scaling.

Reply:
I don't know what you're saying about the scaling Jan :)

Reply:
You talkin' to me? You talkin' to me, Tannen? Well, I'm the only one here. ;)
Sorry - Paula Croal is heading OSIPI 6.1. And she was using this same EPAD data for OSIPI. I thought that besides fixing the JSONs here, we're also fixing them for OSIPI. Wasn't that a task from Henk to you (both)?

The M0 here needs to be re-scaled - so that you can remove the scaling factor fields from Philips. I have already done that and sent the updated M0.nii to OSIPI 6.1. So if you have access to the updated file, you can just copy it here and delete the scaling factors by the commands above and remove then from JSON without applying them.

But if you don't have this M0.nii, we'll have to repeat the re-scaling again (which I would prefer not to do to save time).

I hope that this makes sense now :)

Reply:
I don't have access to any other M0 file. I only know about the separate private OSIPI_TF6.1 repository, but I wasn't working on this. Henk and Beatriz were working on that.

Reply:
Ran it nevertheless once: 9a2ab40